### PR TITLE
add MIT License to the project

### PR DESCRIPTION
**Pull request by saminoorsyed**
Hey there,

I know this likely won't be added yet, but I thought putting in the Licensing info might
be a good step to getting other contributors to work on this project. I wasn't sure who to 
put as the copyright holder for the MIT License, so I put Robert Balicki.

I'll get in touch with Rob about contributing in a more directed way this next week.

--Sami

### CODE REVIEW

**Comment by saminoorsyed** [comment]
Merge conflicts: resolved by defaulting to main and adding licensing info

File: Cargo.toml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[package]
+license = "MIT"
```
Comment (rbalicki2) [inline Cargo.toml:2]: TIL — cargo toml's that define workspaces but not packages (as this one does) don't support the license key.

if you try to compile the project like this, it complains about 

```
error: failed to parse manifest at `/Users/rbalicki/code/isograph/Cargo.toml`

Caused by:
  missing field `name`
  in `package`
```

Can you revert this file change, and then this is good to go and we'll merge it!!!

**Review by rbalicki2 — CHANGES_REQUESTED** [review]
Just one nit re: the base Cargo toml, and then we're good to go!!! Thanks for adding this.

**Comment by saminoorsyed** [comment]
done!

**Review by rbalicki2 — APPROVED** [review]
Awesome, thanks for making these changes!

**Pull request merged**